docusaurus/docs/cloud/projects/collaboration.md

```diff
@@ -38,7 +38,8 @@ To invite a new maintainer to collaborate on a project:
 New maintainers will be sent an email containing a link to click on to join the project. Once a project is shared, avatars representing the maintainers will be displayed in the project dashboard next to the **Share** button, to see how many maintainers collaborate on that project and who they are.
 
 :::tip
-Avatars use GitHub profile pictures, but for pending users only initials will be displayed until the activation of the maintainer account. You can hover over an avatar to display the full name of the maintainer.
+<!-- ? check with Cloud team -->
+Avatars use GitHub or Google profile pictures, but for pending users only initials will be displayed until the activation of the maintainer account. You can hover over an avatar to display the full name of the maintainer.
```

Comment:
Will avatars use the Google profile picture if invited people log in with a Google account?

Reply:
Yes @pwizla Avatar will be updated according to the account that you're logged in with.